src/core/pointcloud/qgscopcpointcloudindex.h

```diff
@@ -51,15 +51,14 @@ class CORE_EXPORT QgsCopcPointCloudIndex: public QgsPointCloudIndex
 
     void load( const QString &fileName ) override;
 
-    bool hasNode( const IndexedPointCloudNode &n ) const override;
-    QList<IndexedPointCloudNode> nodeChildren( const IndexedPointCloudNode &n ) const override;
+    bool hasNode( const QgsPointCloudNodeId &n ) const override;
+    virtual QgsPointCloudNode getNode( const QgsPointCloudNodeId &id ) const override;
```

Comment:
virtual is redundant when override is specified:
```suggestion
    QgsPointCloudNode getNode( const QgsPointCloudNodeId &id ) const override;
```
(elsewhere too)

Reply:
Thanks, I fixed this on all such methods in the diff.